src/app/SBE/search/Search.js

```diff
@@ -170,4 +176,28 @@ const ProcessSearch = ({ sbeCode, process, placeholder }) => {
     </Stack>
   )
 }
+
+const ProcessSearch = () => {
+  const isMobile = useIsMobile()
+  return (
+    <>
+      {isMobile ? (
```

Comment:
{isMobile ? <ProcessSearchMobileView /> : <ProcessSearch />}

where ProcessSearchMobileView Component is encompasses the code from line 185 to line 196.

Reply:
done